meilisearch-release-changelog/v1.0.0.md

```diff
@@ -0,0 +1,257 @@
+      </ul>
+  </details>
+* When calling `GET /index/:uid/documents` and `GET /indexes/:uid/documents/:uid`, some error `code`s have been changed.
+  <details>
+    <summary>Click for details</summary>
+      <ul>
+        <li>Incorrect use of the <code>fields</code> parameter now returns <code>invalid_document_fields</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>limit</code> parameter now returns <code>invalid_document_limit</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>offset</code> parameter now returns <code>invalid_document_offset</code> instead of <code>bad_request</code></li>
+      </ul>
+  </details>
+* When calling `POST /index/:uid/documents` and `PUT /indexes/:uid/documents`
+  <details>
+    <summary>Click for details</summary>
+      <ul>
+        <li>Incorrect use of the <code>primaryKey</code> parameter now returns <code>invalid_index_primary_key</code> instead of <code>bad_request</code></li>
+        <li><code>invalid_document_geo_field</code> has been renamed to <code>invalid_geo_field</code></li>
+      </ul>
+  </details>
+* When calling `GET /index/:uid/search` and `POST /indexes/:uid/search`, some error `code`s have been changed.
+  <details>
+    <summary>Click for details</summary>
+      <ul>
+        <li>Incorrect use of the <code>q</code> parameter now returns <code>invalid_search_q</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>offset</code> parameter now returns <code>invalid_search_offset</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>limit</code> parameter now returns <code>invalid_search_limit</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>page</code> parameter now returns <code>invalid_search_page</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>hits_per_page</code> parameter now returns <code>invalid_search_hits_per_page</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>attributesToRetrieve</code> parameter now returns <code>invalid_search_attributes_to_retrieve</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>attributesToCrop</code> parameter now returns <code>invalid_search_attributes_to_crop</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>showMatchesPosition</code> parameter now returns <code>invalid_search_show_matches_position</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>filter</code> parameter now returns <code>invalid_search_filter</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>sort</code> parameter now returns <code>invalid_search_sort</code> instead of <code>bad_request</code></li>
```

Comment:
```suggestion
        <li>Incorrect use of the <code>filter</code> parameter now returns <code>invalid_search_filter</code> instead of <code>invalid_filter</code></li>
        <li>Incorrect use of the <code>sort</code> parameter now returns <code>invalid_search_sort</code> instead of <code>invalid_code</code></li>
```